### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by martijnlammers** [microsoft/CCF#3047] [Q&A]
I'm currently trying to set up a JS application bundle into an enclave, however I'm struggling to get through the documentation as I am a rather inexperienced programmer and CCF expects quite the amount of prior knowledge. 

I was wondering if the documentation would be getting a more beginner friendly restructuring or whether there are any helpful sources that I could be using to guide me through the process?

**Comment by achamayou** [comment]
I think we would suggest that https://microsoft.github.io/CCF/main/build_apps/js_app_bundle.html is a good starting point to build JS applications for CCF, and https://microsoft.github.io/CCF/main/build_apps/run_app.html a good starting point to run it in a local sandbox.

You may also find the overview useful to understand what CCF provides: https://microsoft.github.io/CCF/main/overview/concepts.html

I'm not quite sure how to start to address your request about making the documentation more beginner friendly, would you perhaps point us in the right direction by telling us what you have found unclear, or sharing the questions you have?
